plugins/by-name/blink-copilot/default.nix

```
@@ -0,0 +1,62 @@
+{ lib, ... }:
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "blink-copilot";
+  package = "blink-copilot";
+
+  maintainers = [ lib.maintainers.khaneliman ];
+
+  description = ''
+    This plugin should be configured through blink-cmp's source settings.
+
+    For example:
+
+    ```nix
+    plugins.blink-cmp = {
+      enable = true;
+      settings.sources = {
```

```suggestion
      settings.sources.providers = {
```

Correct me if I'm wrong, but don't cmp [providers](https://cmp.saghen.dev/configuration/sources.html#providers) get registered under `sources.providers`?

---

Yes, thank you. Just copied the other one and renamed stuff.